### DISCUSSION TITLE
Consume encrypted video stream from AXIS IP Camera

### DISCUSSION THREAD

**Discussion by Lopuch** [AxisCommunications/discussions#250] [General]
Hello,
we need to consume encrypted video stream from AXIS IP Camera.
So far we tried to connect to RTSPS (secured RTSP) stream on port 322 but with no success.

We are able to connect to unencrypted stream by this url for example with ffmpeg:
`ffplay -rtsp_transport tcp -i "rtsp://admin:PASSWORD@IPAddress/axis-media/media.amp"`
But when we add port :322 after IP address, the error 'Invalid data found when processing input' is thrown at us.

We do not stick on ffplay (ffmpeg) - we are open-minded for any solutions which brings us to encrypted video stream.

Any ideas? Thank you :-)

**Comment by vivekatoffice** [comment]
Related link:
[475](https://github.com/orgs/AxisCommunications/discussions/475)
